### PR TITLE
[Docs] PR for "Update client-to-node TLS documentation" and "Document how to rotate self-signed certificates" (Doc-PLAT-2329)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Covers https://yugabyte.atlassian.net/browse/PLAT-2448 and https://yugabyte.atlassian.net/browse/PLAT-2329 (backport the latest changes to 2.4 and 2.6 platform docs)

### CODE REVIEW

File: docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ headerTitle: Enable Encryption in Transit (TLS)
 linkTitle: Enable encryption in transit (TLS)
 description: Use Yugabyte Platform to enable encryption in transit (TLS) on a YugabyteDB universe and connect to clients.
 menu:
-  stable:
+  latest:
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md:7]: Is this correct? Page not showing in menu under stable

File: docs/content/v2.4/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ headerTitle: Enable Encryption in Transit (TLS)
 linkTitle: Enable encryption in transit (TLS)
 description: Use Yugabyte Platform to enable encryption in transit (TLS) on a YugabyteDB universe and connect to clients.
 menu:
-  v2.4:
+  latest:
```
Comment (ddhodge) [inline docs/content/v2.4/yugabyte-platform/security/enable-encryption-in-transit.md:7]: Is this correct? Page not showing in left nav

File: docs/content/v2.6/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ headerTitle: Enable Encryption in Transit (TLS)
 linkTitle: Enable encryption in transit (TLS)
 description: Use Yugabyte Platform to enable encryption in transit (TLS) on a YugabyteDB universe and connect to clients.
 menu:
-  v2.6:
+  latest:
```
Comment (ddhodge) [inline docs/content/v2.6/yugabyte-platform/security/enable-encryption-in-transit.md:7]: Is this correct? Page not showing in left nav

**Review by ddhodge — COMMENTED** [review]
I only see instructions for "How to Rotate Custom CA-Signed Certificates" ticket refers to "We don't document how to rotate self-signed certificates"?

File: docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 4]
```diff
@@ -119,7 +119,7 @@ The following procedure describes how to install certificates on the database no
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md:119]: typo in the link "How to Use Platform-Generate Certificates to Enable TLS" above ->Platform-Generated

File: docs/content/v2.6/yugabyte-platform/security/enable-encryption-in-transit.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/v2.6/yugabyte-platform/security/enable-encryption-in-transit.md:7]: It's only showing in latest. Don't know why. Maybe it'll show when published. Will ask Alex

File: docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/stable/yugabyte-platform/security/enable-encryption-in-transit.md:7]: There is no separate "How to rotate" subsection for every cert type, but it's described in the procedure. (the developer who reviewed the doc when the content was modified for 2.8 agreed that it's fine)

File: docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md:119]: Where? I don't see it

Comment (ddhodge) [reply docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md:119]: line 106

Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md:119]: Fixed

**Pull request merged**